### PR TITLE
Updated Date Kit with new styles and props

### PR DESCRIPTION
**Pull request by christinaatai**
#### Screens

Screen Shot 2020-08-20 at 11 23 45 AM
Screen Shot 2020-08-20 at 11 23 53 AM

#### Breaking Changes

Yes, this will be a breaking change. We are removing size "xs" and renaming the prop "value" to "date" in the React kit to match the prop name in the Rails kit.

If we want to keep everything as similar to what is currently in prod as possible, all Date Kits with size="xs" will should be changed so that size prop is removed, "show_day_of_week"/"showDayOfWeek" is true. All remaining Date Kits (of size "sm"), should be changed so that "show_icon"/"showIcon" and "show_day_of_week"/"showDayOfWeek" is true.

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/NUX-638

#### Checklist:

- [ ] **DEPLOY** Please add the `Milano` label when you are ready for a review
- [x] **SCREENSHOT** Please add a screen shot or two
- [x] **SPECS** Please cover your changes with specs

### CODE REVIEW

File: app/pb_kits/playbook/pb_date/_date.jsx [hunk 1]
```diff
@@ -2,82 +2,85 @@
...
-  </h3>
-)
+const PbDate = (props: PbDateProps) => {
+  const {
+    date,
+    className,
+    showIcon = false,
+    showDayOfWeek = false,
+    alignment = 'left',
+  } = props
```
Comment (kellyeryan) [inline app/pb_kits/playbook/pb_date/_date.jsx:25]: Maybe you could include data, aria, and id props?

File: app/pb_kits/playbook/pb_date/_date.html.erb [hunk 2]
```diff
@@ -2,13 +2,36 @@
...
     data: object.data,
     class: object.classname) do %>
-    <% if object.size == "lg" %>
-      <%= pb_rails("title", props: { text: object.lg_date, size: 3 }) %>
-    <% elsif object.size == "sm" %>
-      <%= pb_rails("icon", props: { icon: "calendar", fixed_width: true }) %>
-      <%= pb_rails("title", props: { tag: "span", text: object.sm_date, size: 4 }) %>
-    <% else %>
-      <%= pb_rails("title", props: { text: object.xs_date, size: 4 }) %>
+
```
Comment (kellyeryan) [inline app/pb_kits/playbook/pb_date/_date.html.erb:5]: Could you add an aria prop here?

File: app/pb_kits/playbook/pb_date/_date.jsx (hunk 1 above)
Comment (coleerikson) [reply app/pb_kits/playbook/pb_date/_date.jsx:25]: Added 👍

File: app/pb_kits/playbook/pb_date/_date.html.erb (hunk 2 above)
Comment (coleerikson) [reply app/pb_kits/playbook/pb_date/_date.html.erb:5]: added.

**Pull request closed without merging**